lib/rebaser-server/src/server/core_loop.rs

```diff
@@ -71,24 +75,27 @@ pub(crate) async fn setup_and_run_core_loop(
     let stream = layer_db.activity().rebase().subscribe_work_queue().await?;
 
     info!("setup complete, entering core loop");
-    core_loop_infallible(ctx_builder, stream, shutdown_watch_rx).await;
+    run_me(ctx_builder, stream, shutdown_watch_rx).await;
     info!("exited core loop");
 
     Ok(())
 }
 
-async fn core_loop_infallible(
+async fn run_me(
```

Comment:
I much prefer my old name for it
<img src="https://media3.giphy.com/media/QJA6xiVxYpmBGXmKR5/giphy.gif"/>

Reply:
I mean, I think your name is probably better! I had meant to refactor it, actually. :)